Fixed a compiler error in swift package

Vignesh-codezyng opened the pull request:
Hello Team, I got a compiler error while using swift package of this repository. I have fixed the error. Kindly review my PR.

Thanks
Vignesh

thomasvl commented:
The files you are changing are generated code, so it would actually be a fix to the generator, not that code.  We already have some other issues on this project and gtm-session-fetcher related to SwiftPM issues, but we haven't really nailed down exactly when things fail so we can put some tests on things to be sure things won't break in the future and work with all packaging systems. If you can provide some more details about how things specific fail, it might help use find the proper generation solution.

Vignesh-codezyng commented:
Hello, The issue was when i tried to use this package for YouTube services I got some errors saying GTLRObject.h file not found. So I fixed it by changing the import path in the 3 files for which I have raised a PR.

The import path I added is,
#import ["../Objects/GTLRObject.h"]

As I changed the import path the package got build and it worked for me.

I have attached a sample below which is working fine when i try to import the package in my project. Kindly refer the sample,

[TestingRestApIClient.zip](https://github.com/google/google-api-objectivec-client-for-rest/files/8123757/TestingRestApIClient.zip)

thomasvl commented:
We just did a 2.0 release where we changed the project layout to use the same sorta layout that Firebase, etc. do, so hopefully this will solve these SwiftPM for folks.
